### PR TITLE
lv: init at 4.51

### PR DESCRIPTION
**Pull request by kayhide**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -2700,6 +2700,11 @@
     email = "info+nix@chmist.com";
     name = "karolchmist";
   };
+  kayhide = {
```
Comment (mmahut) [inline maintainers/maintainer-list.nix]: Please submit this as a separate commit.

**Comment by mmahut** [comment]
@GrahamcOfBorg build lv

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (kayhide) [reply maintainers/maintainer-list.nix]: @mmahut Okay, I have done it! Please take a look.

**Comment by mmahut** [comment]
@GrahamcOfBorg build lv

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -2700,6 +2700,11 @@
...
+    email = "kayhide@gmail.com";
+    github = "kayhide";
```
Comment (aanderse) [inline maintainers/maintainer-list.nix]: @kayhide there is a new requirement that your `github` username be accompanied by a `githubid`. Your `githubid` is [1730718](https://api.github.com/users/kayhide).

Comment (kayhide) [reply maintainers/maintainer-list.nix]: Got it!

**Comment by mmahut** [comment]
@GrahamcOfBorg build lv

File: pkgs/tools/text/lv/default.nix [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+stdenv.mkDerivation rec {
+  pname = "lv";
+  version = "4.51";
+
+  src = fetchurl {
+    url = "mirror://debian/pool/main/l/${pname}/${pname}_${version}.orig.tar.gz";
+    sha256 = "0yf3idz1qspyff1if41xjpqqcaqa8q8icslqlnz0p9dj36gmm5l3";
+  };
+
+  makeFlags = [ "prefix=$(out)" ];
```
Comment (aanderse) [inline pkgs/tools/text/lv/default.nix]: You can use `prefix=${placeholder "out"}` here.

Comment (kayhide) [reply pkgs/tools/text/lv/default.nix]: Done!

**Comment by aanderse** [comment]
@GrahamcOfBorg build lv
@mmahut did you test this by chance?
@kayhide I apologize for the delay.

**Review by aanderse — APPROVED** [review]
Tested and good to go :+1: 
Thanks for your contribution and sorry for the delay @kayhide. Stay safe.

**Comment by kayhide** [comment]
@aanderse Great, thanks!
No worry about the delay, I understand you got a huge amount of issues and PRs.
Thanks for your great work!
